.github/workflows/build.yml

```diff
@@ -8,19 +8,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          submodules: recursive
       - uses: actions/setup-node@v2
-      - uses: cachix/install-nix-action@v13
-      - uses: cachix/cachix-action@v10
+      - uses: onbjerg/foundry-toolchain@v1
         with:
-          name: radicle-drips-hub
-          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+          version: nightly
```

Comment:
Maybe we should stick to a specific nightly version, the one with the `-<hash>` suffix. Otherwise the CI will run a different version every few days.

Reply:
Agree.